apps/easypid/README.md

```diff
@@ -4,25 +4,78 @@
 
 <h1 align="center"><b>Animo EasyPID</b></h1>
 
-This app was created by [Animo Solutions](https://animo.id/) in the context of the [SPRIN-D Funke ‘EUDI Wallet Prototypes’](https://www.sprind.org/en/challenges/eudi-wallet-prototypes/). It serves as a prototype for future wallet providers. For more information on the project reach out to <ana@animo.id>.
+This app was created by [Animo Solutions](https://animo.id/) in the context of the [SPRIN-D Funke ‘EUDI Wallet Prototypes’](https://www.sprind.org/en/challenges/eudi-wallet-prototypes/). It serves as a prototype for future wallet providers and can be tested/used with our [playground environment](https://funke.animo.id/). For more information on the project reach out to <ana@animo.id>. 
 
 During the project an identity wallet and a test relying party was delivered.
 
+
+## Features
 The identity wallet contains the following features, you can see the full flow without running the app in the [Figma design](https://www.figma.com/proto/gBBLERk7lkE27bw8Vm3es4/Funke?show-proto-sidebar=1):
 
-- Onboard user
-- Set up PIN
-- Set up biometrics
-- Authentication using biometrics or PIN
-- Obtain PID from PID provider ([C option](https://gitlab.opencode.de/bmi/eudi-wallet/eidas-2.0-architekturkonzept/-/blob/main/architecture-proposal.md))
-- Obtain PID from PID provider ([B' option](https://gitlab.opencode.de/bmi/eudi-wallet/eidas-2.0-architekturkonzept/-/blob/main/architecture-proposal.md))
-- Present attestations remotely using cross-device QR flow
-- Present attestations remotely using same-device flow
-- About the app
+
+**General App**
+- 🟢 Onboard user
+- 🟢 Set up PIN
+- 🟢 Set up biometrics
+- 🟢 History
+- 🟠 About the app
+- 🟢 Authentication using biometrics or PIN
+
+**Credential Management**
+- 🟢 Credential detail
+- 🟢 Delete QEAA
+- 🟢 Transaction history
+- 🟢 SD-JWT VC Type Metadata
```

Comment:
Type metadata is not supported yet

Reply:
Is it then not a replacement for branding? Becasue we do support branding

Reply:
No it's an extension